test-packages/test-services-openapi/test-service/schema/discriminator-object-entity-workaround-child-only-type.d.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2024 SAP SE or an SAP affiliate company. All rights reserved.
+ *
+ * This is a generated file powered by the SAP Cloud SDK for JavaScript.
+ */
+import type { DiscriminatorObjectEntityWorkaroundParent } from './discriminator-object-entity-workaround-parent';
+/**
+ * Representation of the 'DiscriminatorObjectEntityWorkaroundChildOnlyType' schema.
+ */
+export type DiscriminatorObjectEntityWorkaroundChildOnlyType =
+  DiscriminatorObjectEntityWorkaroundParent;
```

Comment:
[q] This is still confusing to me, didn't we discuss that it should not reference the parent, but rather have properties of the parent?